src/components/createRecipe/createRecipe/CreateRecipe.jsx

```diff
@@ -10,12 +10,11 @@ import Footer from "../../navigation/footerNav/FooterNav";
 
 function App(props) {
 
-  const [page, setPage] = useState(3);
+  const [page, setPage] = useState(4);
```

Comment:
We'll want this to be 1 by default, no? (Or 0, or wherever it starts counting from?)